Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python
+
+ENV DOCKERIZE_VERSION v0.6.1
+
+
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+
+COPY /api.py /app/api.py
+
+COPY /params.py /app/params.py
+
+COPY /.env /app/.env
+
+RUN pip install flask
```

Comment:
i thought you should run as many different commands as you can to cache many comands

Reply:
caching layers is important, but you should be cautious about it, because creating a huge image is worst in term of memory usage
think about it in terms of what is something i change constantly- then this command should be in the end of my dockerfile, so the commands that don't change will save me time in building the image